backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/Autoboxing.kt

```diff
@@ -309,6 +309,24 @@ private class InlineClassTransformer(private val context: Context) : IrBuildingT
         return declaration
     }
 
+    override fun visitCall(expression: IrCall): IrExpression {
+        expression.transformChildrenVoid(this)
+        if (context.shouldOptimize()) {
+            val property = expression.symbol.owner.correspondingPropertySymbol?.owner ?: return expression
+
+            property.parent.also {
```

Comment:
Minor: it might be more appropriate to use `let` here. See https://kotlinlang.org/docs/reference/scope-functions.html#function-selection.